scrapinghub/client.py

```diff
@@ -366,6 +882,29 @@ def get_versioned_cached_store(self, colname):
 
 
 class Collection(object):
+    """Representation of a project collection object.
+
+    Use Project collections methods to get access to Collection.
+
+    Examples:  # noqa
+        >>> foo_store = collections.get_store('foo_store')
+        >>> foo_store.set({'_key': '002d050ee3ff6192dcbecc4e4b4457d7',
+                           'value': '1447221694537'})
+        >>> foo_store.count()
+        1
+        >>> foo_store.get('002d050ee3ff6192dcbecc4e4b4457d7')
+        {'value': '1447221694537'}
+        >>> # iterate over _key & value pair
+        ... list(foo_store.iter())
+        [{'_key': '002d050ee3ff6192dcbecc4e4b4457d7', 'value': '1447221694537'}]
+        >>> # filter by multiple keys
+        >>> # only values for keys that exist will be returned
```

Comment:
It would be better to convert comments to the text

```rst
Filter by multiple keys, only values for keys that exist will be returned::

    >>> list(foo_store.iter(key=['002d050ee3ff6192dcbecc4e4b4457d7', 'blah']))
    [{'_key': '002d050ee3ff6192dcbecc4e4b4457d7', 'value': '1447221694537'}]
```